app/controllers/api/account.php

```diff
@@ -2200,6 +2200,10 @@
         if (empty(App::getEnv('_APP_SMTP_HOST'))) {
             throw new Exception(Exception::GENERAL_SMTP_DISABLED, 'SMTP Disabled');
         }
+        
+        if($user->getAttribute('emailVerification') == true){
```

Comment:
The comparison of a boolean to a `true` value is redundant.

Reply:
fixed